store/sqlstore/supplier_test.go

```diff
@@ -151,6 +153,45 @@ func TestGetDbVersion(t *testing.T) {
 	}
 }
 
+func TestRecycleDBConns(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping recycle DBConns test")
+	}
+	testDrivers := []string{
+		model.DATABASE_DRIVER_POSTGRES,
+		model.DATABASE_DRIVER_MYSQL,
+		model.DATABASE_DRIVER_SQLITE,
+	}
+
+	for _, driver := range testDrivers {
+		t.Run(driver, func(t *testing.T) {
+			settings := makeSqlSettings(driver)
+			supplier := sqlstore.NewSqlSupplier(*settings, nil)
+
+			var wg sync.WaitGroup
+			tables := []string{"Posts", "Channels", "Users"}
+			for _, table := range tables {
+				wg.Add(1)
+				go func(table string) {
+					defer wg.Done()
+					res := supplier.DoesTableExist(table)
```

Comment:
It's not super obvious why this test is testing table existence? Should we just run a hard-coded, manual query if we're trying to open a connection? (The current code suggests that the recycling is somehow tied to table existence.)

Reply:
> Should we just run a hard-coded, manual query if we're trying to open a connection? 

Yeah, I think that makes sense.